### PR TITLE
[doc]Balance stop

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
add `BALANCE STOP` command in doc.

### CODE REVIEW

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/storage-balance.md [hunk 1]
```diff
@@ -164,6 +164,18 @@ Got 8 rows (Time spent: 5074/6488 us)
 
 可以看到 partition 和对应的数据已均衡的分布至各个机器。
 
+## Balance stop
+
+`BALANCE STOP` 命令用于停止已经开始执行的 balance data 计划。如果没有正在运行的 balance 计划，则会返回错误信息。如果有正在运行的 balance 计划，则会返回计划对应的 ID。
```
Comment (critical27) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/storage-balance.md]: BALANCE DATA STOP

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/storage-balance.md [hunk 2]
```diff
@@ -164,6 +164,18 @@ Got 8 rows (Time spent: 5074/6488 us)
...
+
+> 由于每个 balance 计划对应若干个 balance task，`BALANCE STOP` 不会停止已经开始执行的 balance task，只会取消后续的 task，已经开始的 task 将继续执行直至完成。
+
+用户可以在 `BALANCE STOP` 之后输入 `BALANCE DATA $id` 来查看已经停止的 balance 计划状态。
```
Comment (critical27) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/storage-balance.md]: ditto

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/storage-balance.md [hunk 3]
```diff
@@ -164,6 +164,18 @@ Got 8 rows (Time spent: 5074/6488 us)
 
 可以看到 partition 和对应的数据已均衡的分布至各个机器。
 
+## Balance stop
+
+`BALANCE STOP` 命令用于停止已经开始执行的 balance data 计划。如果没有正在运行的 balance 计划，则会返回错误信息。如果有正在运行的 balance 计划，则会返回计划对应的 ID。
+
+> 由于每个 balance 计划对应若干个 balance task，`BALANCE STOP` 不会停止已经开始执行的 balance task，只会取消后续的 task，已经开始的 task 将继续执行直至完成。
```
Comment (critical27) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/storage-balance.md]: ditto

**Comment by dangleptr** [comment]
Jenkins go

**Pull request merged**